[2025-02-02T20:08:08.536+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T20:08:08.598+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_gener_data.stg-load scheduled__2025-02-02T20:07:00+00:00 [queued]>
[2025-02-02T20:08:08.632+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_gener_data.stg-load scheduled__2025-02-02T20:07:00+00:00 [queued]>
[2025-02-02T20:08:08.633+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-02T20:08:08.700+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): stg-load> on 2025-02-02 20:07:00+00:00
[2025-02-02T20:08:08.715+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1549) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T20:08:08.718+0000] {standard_task_runner.py:72} INFO - Started process 1553 to run task
[2025-02-02T20:08:08.720+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_gener_data', 'stg-load', 'scheduled__2025-02-02T20:07:00+00:00', '--job-id', '705', '--raw', '--subdir', 'DAGS_FOLDER/DAG_test_data.py', '--cfg-path', '/tmp/tmp_hjtyd7t']
[2025-02-02T20:08:08.726+0000] {standard_task_runner.py:105} INFO - Job 705: Subtask stg-load
[2025-02-02T20:08:08.860+0000] {task_command.py:467} INFO - Running <TaskInstance: load_gener_data.stg-load scheduled__2025-02-02T20:07:00+00:00 [running]> on host a664bcf281bf
[2025-02-02T20:08:09.591+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_gener_data' AIRFLOW_CTX_TASK_ID='stg-load' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T20:07:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-02T20:07:00+00:00'
[2025-02-02T20:08:09.593+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T20:08:09.865+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:09.980+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T20:08:10.012+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T20:08:10.013+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_gener_data, task_id=stg-load, run_id=scheduled__2025-02-02T20:07:00+00:00, execution_date=20250202T200700, start_date=20250202T200808, end_date=20250202T200810
[2025-02-02T20:08:10.122+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T20:08:10.182+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-02T20:08:10.186+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
